Langs/0415.lang

```diff
@@ -12,8 +12,8 @@ Donate=Wesprzyj
 Settings=Ustawienia
 Check for Updates=Sprawdź aktualizacje
 Your Windows 11 Compatibility Results are Below=Twoje wyniki kompatybilności Windows 11 są poniżej
-Now Reach WhyNotWin11 via https://www.whynotwin11.org/=Teraz WhyNotWin11 jest dostępne na https://www.whynotwin11.org/
-Results Based on Currently Known Requirements!=Wyniki na podstawie obecnie znanych wymagań!
+Now Reach WhyNotWin11 via https://www.whynotwin11.org/=Teraz program WhyNotWin11 jest dostępny na https://www.whynotwin11.org/
+Results Based on Currently Known Requirements!=Wyniki na podstawie obecnie znanych wymagań !
```

Comment:
Nie podobaja mi sie te spacje przed znakami interpunkcyjnymi. Co o tym sadzicie? @BxOxSxS @kp1971

Reply:
@micwoj92 jako że trochę rozmawialiśmy na Au3Forum, pozwolę sobie na uwagę co do twojej znajomości języka Polskiego.
Niestety ale masz już przyzwyczajenia obcojęzyczne i widać, że nie zdawałeś matury w Polsce.

Ja moją zdawałem 23 lata temu więc mogę się mylić ;) ale mam nadzieję, że @BxOxSxS i @kp1971 potwierdzą moje zmiany.

Reply:
Niestety wydaje mi sie, ze jestes w bledzie
https://sjp.pwn.pl/zasady/Stosowanie-lub-pomijanie-spacji-miedzy-literami-a-znakami-interpunkcyjnymi;4836490.html

Reply:
Matura z interpunkcji internetowej? Brzmi ciekawie. Z tego co wiem w internecie od przez każdym znakiem interpunkcyjnym nie ma spacji. Potwierdzają to liczne artykuły pisanych przez profesjonalnych dziennikarzy oraz strony rządowe

Reply:
> Z tego co wiem w internecie od przez każdym znakiem interpunkcyjnym nie ma spacji. 

Na sjp sa podane 3 wyjatki ;p

Reply:
Również mam wątpliwości co do dodania frazy "program". Niepotrzebnie wydłuża to wyrażenie

Reply:
A ta moja propozycja wynika  z tego że nigdy nie lubiłem skrótów myślowych, wolę używać dookreśleń, z uwagą aby nie stosować pleonazmów.
Poza tym miejsce jest więc wydłużenie wyrażenia nie przeszkadza.